plugins/@grouparoo/snowflake/src/lib/export/exportRecords.ts

```diff
@@ -0,0 +1,507 @@
+import {
+  BatchConfig,
+  BatchExport,
+  BatchGroupMode,
+  BatchMethodAddToGroups,
+  BatchMethodCreateByForeignKeyAndSetDestinationIds,
+  BatchMethodDeleteByDestinationIds,
+  BatchMethodFindAndSetDestinationIds,
+  BatchMethodGetClient,
+  BatchMethodNormalizeForeignKeyValue,
+  BatchMethodNormalizeGroupName,
+  BatchMethodRemoveFromGroups,
+  BatchMethodUpdateByDestinationIds,
+  BatchSyncMode,
+  buildBatchExports,
+  exportRecordsInBatch,
+  GroupNameListMap,
+} from "@grouparoo/app-templates/dist/destination/batch";
```

Comment:
this is the first SQL destination I think to use the destination batch helpers!  While a bit more confusing, the other SQL destinations only work on one row at a time... so this is an improvement!

Reply:
Yeah!